### ISSUE TITLE
[Bug]: path.isTSClassImplements is not a function

### ISSUE THREAD

**Issue by coderaiser** [babel/babel#17006]
### 💻

- [ ] Would you like to work on a fix?

### How are you using Babel?

Programmatic API (`babel.transform`, `babel.parse`)

### Input code

```
export default function (babel) {
  const { types: t } = babel;
  
  return {
    name: "ast-transform", // not required
    visitor: {
      VariableDeclaration(path) {
        console.log(path.isTSClassImplements());
        console.log(path.isTSParameterProperty());
      }
    }
  };
}
```

### Current and expected behavior

expected function to exist

https://astexplorer.net/#/gist/e590cbc299c0bc311999e7aeb9dec53f/a2141235cdd60e61663c532073ef5934f4022e07

In a similar to `path.isIdentifier()` way

### Environment

Babel 7, 8

**Comment by JLHwung** [comment]
I think it works as intended: `TSClassImplements` is Babel 8 AST, it should not exist in Babel 7. https://github.com/babel/babel/pull/16731

**Comment by coderaiser** [comment]
In Babel v8 same behaviour

**Comment by coderaiser** [comment]
Also I have:
<img width="727" alt="image" src="https://github.com/user-attachments/assets/5b19ad98-fb42-4bce-b173-49403de83ea8">

For this code:

```js
import {isTSClassImplements} from '@babel/types';

console.log(isTSClassImplements);
```

This is latest Babel 8

**Comment by coderaiser** [comment]
When I run `BABEL_8_BREAKING=true make generate-type-helpers` in `main` brunch of the repo I see it is generated:

<img width="423" alt="image" src="https://github.com/user-attachments/assets/f1acd552-57a4-4443-aaea-213e5a6af785">

**Comment by JLHwung** [comment]
Fixed in `@babel/types` [v8.0.0-alpha.14](https://www.npmjs.com/package/@babel/types/v/8.0.0-alpha.14)
